TicTacToeGame/Controllers/BoardController.cs

```diff
@@ -37,19 +37,73 @@ public string Get(int id)
         }
```

Comment:
Please refrain from using 'i','j','x','y' as variable names and loop variables. 'rowIndex', 'colIndex' look more appropriate.

Reply:
Updated. Please check @darpansaraf